src/cluster_legacy.c

```diff
@@ -5314,6 +5317,11 @@ static int clusterNodeCronHandleReconnect(clusterNode *node, mstime_t now) {
     }
 
     if (node->link == NULL) {
+        if (now - node->last_connect_attempted <= 1000 || *cluster_conn_attempts > MAX_CONNECTION_ATTEMPTS_PER_CRON) {
```

Comment:
Add or reuse a macro for wait time instead of hardcoding it as `1000`.

Reply:
Also, maybe we can avoid this condition here and let this method just perform reconnection in a guaranteed manner. We can move this logic to the caller. WDYT?